src/ddl/migrations/covid_hosp_state_split_tables.sql

```diff
@@ -130,19 +9,16 @@ CREATE TABLE `covid_hosp_state_daily` (
   KEY `date_by_issue_and_state` (`issue`, `state`, `date`),
   -- for fast lookup of all states for a given date and issue
   KEY `state_by_issue_and_date` (`issue`, `date`, `state`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+) ENGINE=InnoDB DEFAULT CHARSET=utf8
+SELECT * FROM covid_hosp_state_timeseries;
+ALTER TABLE covid_hosp_state_daily MODIFY id INT NOT NULL AUTO_INCREMENT;
```

Comment:
its worth a comment here explaining why we are executing this statement.  same in the definition file.